RELEASE.md

```diff
@@ -70,6 +70,7 @@ For a **MAJOR** or **MINOR** release:
 - Run the `make build-install-yaml` command which will generate install files in the `release/` directory.
   Attach these files to the GitHub release.
 - Update the `README.md` and `site-src/guides/index.md` files to point links and examples to the new release.
+- Update the `site-src/implementation-table.md` to `site-src/implementation-table-<oldVersion>.md`. Add the path to the `past_versions` variable in `hack/mkdocs-generate-conformance.py`.
```

Comment:
I think we probably want to add some kind of fixed and predictable timeline for this. We can't expect implementations to support the latest release on day 1, but maybe we switch over the default link 1 month after the release or after we've received 3 reports, whichever comes first?

/cc @shaneutt @youngnick @mlavacca

Reply:
I like the suggestion that we either do 1month max, or at least 3 reports, that seems solid. We can adjust from there if anyone has complaints.

Reply:
Yes, +1 on 1 month or 3 reports, whichever happens first.

Reply:
Sounds good to me as well 👍